school-manager-server/src/main/resources/application.yml

```diff
@@ -0,0 +1,19 @@
+server:
+  port: 8080
+spring:
+  datasource:
+    url: jdbc:mysql://${MYSQL_HOST:localhost}:3306/HW2
+    username: root
+    password: 1qazXSW@3edc
+    driver-class-name: com.mysql.cj.jdbc.Driver
```

Comment:
You should not push your database login and password into repo. 
Use env variables or additional conf file instead. 
Example: 
username: ${DB_USERNAME:username} - in case of env variables override during app start
password: ${DB_PASSWORD:password} 
or 
spring.config.additional-location - set your password and login in separate config. This conf file should be used locally. On production it will be copied to classpass before compilation.